### PR TITLE
Draft out a PR template

### PR DESCRIPTION
**Pull request by klappradla**
As we tend to forget to update the `CHANGELOG.md`, I added a reminder to a global PR template.

The template is a bit work in progress, as I did not really know if there's something else important we want to put there...

Any suggestions?

### CODE REVIEW

**Review by andreasknoepfle — APPROVED** [review]
Awesome!

**Comment by andreasknoepfle** [comment]
Btw @max: Let's just start simple with this for now and add new things if we feel we need them.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!--
+PR checklist:
+- Does this introduce breaking changes?
```
Comment (tessi) [inline .github/PULL_REQUEST_TEMPLATE.md]: hmm 🤔 would be great if the checklist contained actionable items. i don't know if this is actionable. i answer this with yes/no and then what?

Do you mean something like `describe breaking changes in the PR/commit body`?

Comment (klappradla) [reply .github/PULL_REQUEST_TEMPLATE.md]: No. This is more a successor to the next question. Probably makes sense to just remove this point and only have the one about the CHANGELOG.md 🤔

**Review by nickgnd — APPROVED** [review]
Neat 👍

**Comment by klappradla** [comment]
What just came to my mind: is `2.0.0` correct 🤔

**Pull request merged**